build/webpack.base.conf.js

```diff
@@ -12,7 +12,7 @@ module.exports = {
     alias: {
       'src': path.resolve(__dirname, '../src'),
       'src-docs': path.resolve(__dirname, '../src-docs'),
-      'focus-trap': path.resolve(__dirname, 'node_modules/focus-trap/index.js')
+      'focus-trap': path.resolve(__dirname, '../node_modules/focus-trap/index.js')
```

Comment:
This wasn't allowing the `npm run build` or `npm run dev` to execute properly, had to update the paths to work